packages/contracts/src/components/ZoneComponent.sol

```diff
@@ -4,9 +4,9 @@ pragma solidity >=0.8.0;
 import "std-contracts/components/Uint256Component.sol";
 import { ZoneEnum } from "libraries/MSTypes.sol";
 
-uint256 constant ID = uint256(keccak256("component.Position"));
+uint256 constant ID = uint256(keccak256("component.Zone"));
 
-contract PositionComponent is Uint256Component {
+contract ZoneComponent is Uint256Component {
```

Comment:
why is zone replacing position?

Reply:
zone is not replacing position I copied the position component but forgot to change the name to zone. Zone refers to the 4 spots on a location and is tied to the character.